src/main/java/us/kbase/sdk/compiler/RunCompileCommand.java

```diff
@@ -123,8 +123,6 @@ public Map<String, KbModule> parseInclude(String includeLine) throws KidlParseEx
                 }
             }
         }
-        if (javaServerSide)
-            javaClientSide = true;
```

Comment:
Do python and java versions now both act the same wrt: arguments supplied and clients generated?